.github/workflows/docs.yml

```diff
@@ -20,13 +20,15 @@ jobs:
     - name: Install dependencies
       working-directory: l5kit
       run: |
-        python -m pip install --upgrade pip
-        pip install -r requirements.txt
+        pip install --upgrade pip
+        pip install pipenv
+        pipenv install --dev
+        pip install .
```

Comment:
what is this last line for?

Reply:
we read the version from the `__init__` in l5kit, so we need it to be installed here..

Reply:
not sure I understand. What is this installing?

Reply:
l5kit itself from the `setup.py`

Reply:
but if you add l5kit to the pipfile there is no need for this, right?

Reply:
ok that makes sense, should I add it in editable mode then?

Reply:
yep, I would add it there so that pipenv manages everything. There should't be the need to run any pip command